runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierConfig.java

```diff
@@ -89,4 +90,34 @@ default int getPrayerThreshold()
 	{
 		return 0;
 	}
+
+
+	@ConfigItem(
+		keyName = "message",
+		name = "Notify on message",
+		description = "Configures if notification by messages should be enabled.",
+		position = 6
+	)
+	default boolean message()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = "messageNotification",
+		name = "Notified messages",
+		description = "Configures specifically which messages to notify you of. Format: (message), (message)",
+		position = 7
+	)
+	default String getMessageNotification()
+	{
+		return "";
+	}
+
+	@ConfigItem(
+		keyName = "messageNotification",
+		name = "",
+		description = ""
+	)
+	void setMessageNotification(String key);
```

Comment:
Will this be used by something?

Reply:
I was planning to add onto this to have a hotkey modifier similar to ground items for marking chat messages. But I can always add this at a later date. I will remove for now.